nixnet/database/_dbc_signal_value_table.py

```diff
@@ -97,6 +97,6 @@ def _value_table(self):
         # table_string is of the format 'value1, key1, value2, key2, ...'
         # convert to a dict: { 'key1': int('value1'), 'key2': int('value2'), ... }
         table_dict = dict(
-            (key, value)
+            (key, int(value))
```

Comment:
Based on testing done for #229, we found that this cast is needed because the driver converts value to an int but returns it in a string format (we tested signal values of type string and float).

It may be worth calling it out as a comment.